esp/src/test-ui/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd ./tests
+for file in *.class; do
+    echo "$(basename "$file")"
+    java "${file%.*}" $1
```

Comment:
This should tally the exit codes (error counts) and return it.

Reply:
Did you expect this kind of output:
```
sudo docker run eclwatch-test:latest  http://192.168.1.54:8010
Activities.class
Feb 02, 2021 1:46:59 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Pass
Pass
Pass
Activities2.class
Feb 02, 2021 1:47:08 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Pass
Pass
Fail
```

Reply:
Yes - but in the newer version it keeps a tally of all the failures (non zero exit codes) and returns that from the script (non zero exit code).  So it will be easier for you to spot.

The Pass/Fail text is from the Java Test so theoretically out of my control.

Reply:
Would the exit code be same as the number of failed tests?

Reply:
Yes (assuming the Java tests adhere to that convention)